### ISSUE TITLE
LG Thinq HA Core integration

### ISSUE THREAD

**Issue by rrooggiieerr** [ollo69/ha-smartthinq-sensors#817]
Just FYI, there seems to be a LG Thinq HA Core integration in the making

https://github.com/home-assistant/core/tree/lg_thinq/homeassistant/components/lg_thinq
https://github.com/home-assistant/home-assistant.io/blob/lg_thinq/source/_integrations/lg_thinq.markdown

It uses this library
https://github.com/thinq-connect/pythinqconnect

And an official API
https://thinq.developer.lge.com/en/cloud/

**Comment by ollo69** [comment]
Interesting, so I suppose that when ready will replace this integration.
I will take a look, thanks for informing.

**Comment by w-marco** [comment]
Fwiw I loaded it in as a custom component today to play around and especially for dryers and washers it's still _really_ barebones, though we can't know at which state they will release it. But at least for quite a while I can't imagine that it can fully replace this integration.

But maybe it could serve either as inspiration or work from this integration could be merged into core. Or they could just co-exist (see Nuki Core and Nuki-Ng custom_component both living in co-existence). 

Maybe the auth via Personal Access Token instead of User/Password would be interesting for this integration?

**Comment by felipecrs** [comment]
It should be released to stable in HA 2024.11.0 btw.

**Comment by felipecrs** [comment]
It's crazy to see the lot of work spent here just being redone, but I believe this integration is being sponsored by LG. I don't know why the user would be named that way otherwise.

![image](https://github.com/user-attachments/assets/16e56364-c48a-4a5c-9bde-212bc8217865)

**Comment by rrooggiieerr** [comment]
> It should be released to stable in HA 2024.11.0

I saw that too. 

> It's crazy to see the lot of work spent here just being redone

Exactly my thoughts, they could at least have contacted the developer for collaboration. Having said that, they seem to have developed a decent API and library with push updates using mqtt instead of polling.

**Comment by ollo69** [comment]
> Exactly my thoughts, they could at least have contacted the developer for collaboration

Yes, it would have been nice. But I like to think that all the work done here has at least been inspiring. I am always available to collaborate with HA team if necessary.

**Comment by crazyserver** [comment]
I don't like when this kind of things happen, Open source is our power!

However I want to share that the new official integration completely integrates vacuum cleaners! I'm very happy to see that!

Thanks @ollo69 for all your time dedicated to this project and other projects!

**Comment by NewsGuyTor** [comment]
This is both weird and cool. It is clearly being developed by LG itself (most commits are co-authored by two Korean persons with name@lge.com emails). That is pretty cool by itself, when big brands like LG invest in HA integrations it's a pretty promising sign for the future of HA.

On the other hand it is quite weird that @ollo69 hasn't been told about it/consulted at all. 

Anyway, thanks for all your work @ollo69! It's gotta be both weird and inspiring to see a big brand like LG realize the value of your work so much that they decide to invest company resources into developing their own official integration, yet perhaps also slightly melancholic.

**Comment by w-marco** [comment]
That said, if they don't add a big commit in this or next release, the integration still vastly lacks behind this one.

I have a dozen sensors and full control over remote start for my washer with this custom integration while the official one still has barely any sensors, does not support setting temperature and such (whereas there's a working PR here that does) and so on. So for the foreseeable future I can't see myself switching.

**Comment by bschatzow** [comment]
The core version in beta only has 9 sensors vs this one that has 28 for my washer.

**Comment by bschatzow** [comment]
> > Exactly my thoughts, they could at least have contacted the developer for collaboration
> 
> Yes, it would have been nice. But I like to think that all the work done here has at least been inspiring. I am always available to collaborate with HA team if necessary.

I have raised an issue with the new builtin one in the beta discord channel. The new integration is missing many useful sensors.

**Comment by NukeThemTillTheyGlow** [comment]
I'm on version 0.40.3 and through various updates a few of the sensors are no longer displaying properly and before trying to update and fix things, I was wondering should I just delete the current integration and switch to the new?  I installed manually about 9 months ago, its just for a washer and dryer.  Only really use it for status checking, when things are done, but I also like to be on a fully maintained version too.   Any advice?

**Comment by NukeThemTillTheyGlow** [comment]
> I'm on version 0.40.3 and through various updates a few of the sensors are no longer displaying properly and before trying to update and fix things, I was wondering should I just delete the current integration and switch to the new? I installed manually about 9 months ago, its just for a washer and dryer. Only really use it for status checking, when things are done, but I also like to be on a fully maintained version too. Any advice?

Checking in

**Comment by NewsGuyTor** [comment]
> > I'm on version 0.40.3 and through various updates a few of the sensors are no longer displaying properly and before trying to update and fix things, I was wondering should I just delete the current integration and switch to the new? I installed manually about 9 months ago, its just for a washer and dryer. Only really use it for status checking, when things are done, but I also like to be on a fully maintained version too. Any advice?
> 
> 
> 
> Checking in

I don't see how tgis is related to the official LG integration we discuss here, you should make a new issue.

**Comment by Waank1** [comment]
Looking for a little assistance.  I recently installed the LG ThinQ core integration.  (I went to the LG site and downloaded the Token then installed the Integration.)  I have an LG LTE4815ST Stove.  Previously I was using the custom integration from HACS and it contained approx 22 entities.  The core integration contains appro 11.  Am I missing something?

**Comment by felipecrs** [comment]
I don't think you're missing anything. You can even keep using this custom integration if you wanted all those entities.

**Comment by mhalano** [comment]
I tried with an AC, and it seems there are fewer options and the climate entity platform has fewer features.

**Comment by francosax** [comment]
Hello @ollo69 and others guys,
I tried this integration from LG, but it is very raw...for example the HVAC ambient temperature is not available as a variable! and air purifier switch not available at all!
I was obliged to try this because of the Ollo69 integration is not working anymore with the HA core 2025.4. But at the moment I restored a backup with previous HA core... Do you think I have to open a ticket for the last problem?

thank you
Franco

**Comment by oerix** [comment]
Hi @ollo69, I've noticed that last update was in January, and there are countless of issues have been created since then. Has this integration been abandoned in favour for the core integration, or are you working with the core team?
